benchmark/feldera-sql/benchmarks/nexmark/table.sql

```diff
@@ -74,3 +74,23 @@ CREATE TABLE bid (
         }}
     }}
 ]');
+CREATE TABLE side_input (
+  date_time TIMESTAMP,
+  key BIGINT,
+  value VARCHAR
+) WITH ('connectors' = '[{{
+  "transport": {{
+      "name": "datagen",
+      "config": {{
+        "plan": [
+          {{
+            "limit": 100,
+            "fields": {{
+              "date_time": {{ "range": [1724444408000, 4102444800000] }},
+              "key": {{ "range": [0, 100] }}
```

Comment:
since you have limit: 100, you don't need to specify anything for key here, should fill it from 0..99 without this line

Reply:
that's ok, I am not sending another commit for this change
thanks for the hint